src/app/core/services/navigation.service.ts

```diff
@@ -81,6 +84,14 @@ const navigationService = {
       errorService.reportError(error);
     }
   },
+  setWorkspaceFromParams(workspaceThunks, dispatch: AppDispatch): void {
+    const params = new URLSearchParams(window.location.search);
+    const currentWorkspaceUuid = params.getAll('workspaceid');
+    currentWorkspaceUuid.length === 1 &&
```

Comment:
maybe could extract the conditional `currentWorkspaceUuid.length === 1` to a const in order to give a proper name to understand the reason

Reply:
I think it's clearer with this change